apps/api/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.9
+
+COPY . .
```

Comment:
Suggestion: Copying source files as the first step defeats the purpose of the build cache. We can better use build cache by copying source code towards the end after installing relevant dependencies. See suggestion below on the line for `pip install ...`